typings/index.d.ts

```diff
@@ -147,8 +145,8 @@ declare module 'discord.js' {
 		public user: ClientUser | null;
 		public users: UserStore;
 		public readonly voiceConnections: Collection<Snowflake, VoiceConnection>;
+		public voice: ClientVoiceManager;
```

Comment:
This property is nullable.
```suggestion
		public voice: ClientVoiceManager | null;
```